osm2streets/src/lib.rs

```diff
@@ -29,9 +29,11 @@ mod geometry;
 mod ids;
 mod intersection;
 mod lanes;
+mod marking;
```

Comment:
Stray thought for later: marking, paint, output, and render maybe belong lumped together in a submodule at some point, if these keep growing